server/package.json

```diff
@@ -57,4 +57,4 @@
     "url": "https://github.com/bravetechnologycoop/BraveButtons/issues"
   },
   "homepage": "https://github.com/bravetechnologycoop/BraveButtons"
-}
+}
```

Comment:
Looks like the newline at the end of this file was removed. Could you please put it back in? I ask this for two reasons:
1. Then there isn't this misleading diff showing up in GitHub
2. If you needed to add something onto the end of the file, then GitHub would also think that the last line changed because you needed to add a newline to it, which is also a little annoying when looking at the GitHub diffs.